File: src/main/java/io/zeitwert/server/config/crnk/DefaultQueryPathResolver.java

package io.zeitwert.server.config.crnk;

public class DefaultQueryPathResolver extends io.crnk.core.queryspec.internal.DefaultQueryPathResolver {

	public DefaultQueryPathResolver() {
		super();
		// This is done because filter does not respect @JsonProperty annotation
		this.setMapJsonNames(false);
	}
}

File: fm-server/src/main/java/io/zeitwert/fm/server/config/crnk/DefaultQueryPathResolver.kt
package io.zeitwert.fm.server.config.crnk

import io.crnk.core.queryspec.internal.DefaultQueryPathResolver

class DefaultQueryPathResolver : DefaultQueryPathResolver() {
	init {
		// This is done because filter does not respect @JsonProperty annotation
		this.mapJsonNames = false
	}
}
